tools/Azure.Mcp.Tools.FunctionApp/src/Commands/FunctionApp/FunctionAppGetCommand.cs

```diff
@@ -17,7 +18,7 @@ public sealed class FunctionAppGetCommand(ILogger<FunctionAppGetCommand> logger)
     private const string CommandTitle = "Get Azure Function App Details";
     private readonly ILogger<FunctionAppGetCommand> _logger = logger;
 
-    private readonly Option<string> _optionalFunctionAppNameOption = FunctionAppOptionDefinitions.OptionalFunctionApp;
+    private readonly Option<string> _functionAppNameOption = FunctionAppOptionDefinitions.FunctionApp;
```

Comment:
Not this PR but we should be consistent on whether this pattern should or should not be used. Based on my last comment, I'm against this local readonly duplication of public / internal accessible readonly definitions.